applications/integrationtests/mechanics/GradientDamage.cpp

```diff
@@ -137,11 +141,12 @@ BOOST_AUTO_TEST_CASE(Integrand)
 
 BOOST_AUTO_TEST_CASE(Integrand2D)
 {
-    /**
-     * This just evaluates the gradient and hessian once
-     * to make sure all the dimensions in the matrix calculations
-     * match.
-     */
+    auto binary = boost::unit_test::framework::master_test_suite().argv[0];
+    boost::filesystem::path binaryPath = std::string(binary);
+    binaryPath.remove_filename();
+    std::string meshFile = binaryPath.string() + "/meshes/Holes.msh";
```

Comment:
Why create this `binary` variable if you don't use it? Just do
~~~cpp
boost::filesystem::path binaryPath =
    boost::unit_test::framework::master_test_suite().argv[0];
~~~

Reply:
Totally right. I mindlessly copied that from somewhere...